views/pages/add-experience.ejs

```diff
@@ -0,0 +1,95 @@
+<!-- Navbar -->
+<%- include('../partials/header')-%>
+<!-- Page Content Starts -->
+
+<div class="addexperience">
+    <div class="addstudentpic">
+        <source srcset="images/BSJ-logo.png" type="image">
+            <img src="images/BSJ-logo.png" class="img-fluid" alt="Responsive image">  
+    </div>
+
+    <div class="addstudentName">
+        <h2>Student Name</h2>
+    </div>
+    
+    <div class="title-addexperience">
+        <h2>Add an Experience</h2>
+    </div>
+    <!--add current organization starts here-->
+    <div class="dropdown-current-organization">
+        <h2>Add current Organization</h2>
+            <button class="bg btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Choose an Organization
+            </button>
+
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item" href="#">Option 1</a>
+            <a class="dropdown-item" href="#">Option 2</a>
+            <a class="dropdown-item" href="#">Option 3</a>
+        </div>
+
+        <div class="adding-neworganization">
+            <button type="button" class="btn btn-secondary">Add a new Organization</button> 
```

Comment:
We will need to add JS to this button. When the button is clicked, the input field will appear. This will need to be done to the other 2 boxes as well.